src/python/pants/core/target_types.py

```diff
@@ -52,11 +52,11 @@ class Resources(Target):
 # -----------------------------------------------------------------------------------------------
 
 
-class GenericTarget(Target):
+class TargetAlias(Target):
     """A generic target with no specific target type.
 
     This is useful for aggregate targets: https://www.pantsbuild.org/target_aggregate.html.
     """
```

Comment:
Stale, and this gets outputted in `./pants target-types`. Maybe something like this for now:


> An alias for multiple other targets.
>
> Add targets that you want to alias to the `dependencies` field. Then, when you run a Pants command on this target, Pants will substitute in those targets. For example, `./pants test $target_alias` will run tests on all of the `dependencies`.